src/command_modules/azure-cli-iotcentral/azure/cli/command_modules/iotcentral/_help.py

```diff
@@ -25,10 +25,10 @@
         - name: Create an IoT Central application with the free pricing tier F1, in the region of the resource group.
           text: >
             az iotcentral app create --resource-group MyResourceGroup --name MyApp --subdomain myapp
-        - name: Create an IoT Central application with the standard pricing tier S1 in the 'westus' region.
+        - name: Create an IoT Central application with the standard pricing tier S1 in the 'westus' region, based on the iotc-demo template.
           text: >
             az iotcentral app create --resource-group MyResourceGroup --name MyApp --sku S1 --location westus
-            --subdomain myapp
+            --subdomain myapp --template iotc-demo@1.0.0
```

Comment:
Should we default the template to "Contoso"?

Reply:
when the user doesn't specific a template, the server will default the app. Do you mean for this example?

Reply:
it might be difficult to create an actual blank app in that case, to override the default something like  `--template blank` would be needed

Reply:
I see, default to blank template sounds good, and we keep the consistency with all other languages.
Can we add the three template options in the help? So users can see the text they can use.